@olamy there's a duplicate class checker that's preventing this PR from proceeding.

This particular set of duplicates have been present since 12.0.0 release.

```
[WARNING] Found duplicate and different resources in [org.eclipse.jetty.toolchain:jetty-schemas:5.2, org.eclipse.jetty.toolchain:jetty-servlet-api:4.0.6]:
[WARNING]   javax/servlet/resources/j2ee_1_4.xsd
[WARNING]   javax/servlet/resources/j2ee_web_services_1_1.xsd
[WARNING]   javax/servlet/resources/j2ee_web_services_client_1_1.xsd
[WARNING]   javax/servlet/resources/javaee_5.xsd
[WARNING]   javax/servlet/resources/javaee_6.xsd
[WARNING]   javax/servlet/resources/javaee_7.xsd
[WARNING]   javax/servlet/resources/javaee_8.xsd
[WARNING]   javax/servlet/resources/javaee_web_services_1_2.xsd
[WARNING]   javax/servlet/resources/javaee_web_services_1_3.xsd
[WARNING]   javax/servlet/resources/javaee_web_services_1_4.xsd
[WARNING]   javax/servlet/resources/javaee_web_services_client_1_2.xsd
[WARNING]   javax/servlet/resources/javaee_web_services_client_1_3.xsd
[WARNING]   javax/servlet/resources/javaee_web_services_client_1_4.xsd
[WARNING]   javax/servlet/resources/permissions_7.xsd
[WARNING]   javax/servlet/resources/web-app_2_2.dtd
[WARNING]   javax/servlet/resources/web-app_2_3.dtd
[WARNING]   javax/servlet/resources/web-app_2_4.xsd
[WARNING]   javax/servlet/resources/web-app_2_5.xsd
[WARNING]   javax/servlet/resources/web-app_3_0.xsd
[WARNING]   javax/servlet/resources/web-app_3_1.xsd
[WARNING]   javax/servlet/resources/web-app_4_0.xsd
[WARNING]   javax/servlet/resources/web-common_3_0.xsd
[WARNING]   javax/servlet/resources/web-common_3_1.xsd
[WARNING]   javax/servlet/resources/web-common_4_0.xsd
[WARNING]   javax/servlet/resources/web-fragment_3_0.xsd
[WARNING]   javax/servlet/resources/web-fragment_3_1.xsd
[WARNING]   javax/servlet/resources/web-fragment_4_0.xsd

...

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:47 min
[INFO] Finished at: 2023-12-13T05:45:20Z
[INFO] ------------------------------------------------------------------------
[INFO] Saving cache report on build completion
[INFO] Saved to remote cache http://nginx-cache-service.jenkins.svc.cluster.local:80/v1/org.eclipse.jetty/jetty-project/796de4cd-f318-46fe-a00a-2f8089b25b9b/build-cache-report.xml
[ERROR] Failed to execute goal org.basepom.maven:duplicate-finder-maven-plugin:2.0.0:check (default) on project jetty-ee8-apache-jsp: Found duplicate classes/resources! -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.basepom.maven:duplicate-finder-maven-plugin:2.0.0:check (default) on project jetty-ee8-apache-jsp: Found duplicate classes/resources!
```

--- a/jetty-ee8/pom.xml
+++ b/jetty-ee8/pom.xml
@@ -49,7 +49,7 @@
     <jakarta.websocket.api.version>1.1.2</jakarta.websocket.api.version>
     <javax.mail.glassfish.version>1.4.1.v201005082020</javax.mail.glassfish.version>
     <jetty.servlet.api.version>4.0.6</jetty.servlet.api.version>
-    <jsp.impl.version>9.0.83</jsp.impl.version>
+    <jsp.impl.version>9.0.83.1-SNAPSHOT</jsp.impl.version>
     <modify-sources-plugin.version>1.0.9</modify-sources-plugin.version>
     <sonar.skip>true</sonar.skip>
     <weld.version>3.1.9.Final</weld.version>

--- a/jetty-ee8/pom.xml
+++ b/jetty-ee8/pom.xml
@@ -49,7 +49,7 @@
     <jakarta.websocket.api.version>1.1.2</jakarta.websocket.api.version>
     <javax.mail.glassfish.version>1.4.1.v201005082020</javax.mail.glassfish.version>
     <jetty.servlet.api.version>4.0.6</jetty.servlet.api.version>
-    <jsp.impl.version>9.0.83.1-SNAPSHOT</jsp.impl.version>
+    <jsp.impl.version>9.0.83.1</jsp.impl.version>
     <modify-sources-plugin.version>1.0.9</modify-sources-plugin.version>
     <sonar.skip>true</sonar.skip>
     <weld.version>3.1.9.Final</weld.version>

